src/appengine/handlers/external_update.py

```diff
@@ -45,9 +49,41 @@ def _mark_errored(testcase, revision, error):
       testcase, revision, message=message)
 
 
-def handle_update(testcase, revision, stacktrace, error):
+def handle_update(testcase, revision, stacktraces, error, protocol_version):
   """Handle update."""
-  logs.log('Got external update for testcase.', testcase_id=testcase.key.id())
+
+  def is_still_crashing(st_index, stacktrace):
+    """
```

Comment:
Nit: we generally start the doctring on the same line as """.

e.g.

```
"""Check if the given stacktrace......"""
```

Reply:
Fixed in [4febc0c](https://github.com/google/clusterfuzz/pull/2555/commits/4febc0c67d15b22ddaa8680818321a3e535470c3).